smdebug/profiler/algorithm_metrics_reader.py

```diff
@@ -4,6 +4,10 @@
 import json
 import os
 
+# Third Party
+import tensorflow as tf
```

Comment:
what if this test is running in a pytorch container?

Reply:
you're right. reverting the change from this file